cmd/honeydipper/configcheck.go

```diff
@@ -53,6 +53,29 @@ func runConfigCheck(cfg *config.Config) int {
 		}
 	}
 
+	builtInContexts := map[string]bool{
+		"*":       true,
+		"_events": true,
+	}
+
+	contextErrors := false
+	for contextName, contextValue := range cfg.DataSet.Contexts {
+		for wfName, _ := range contextValue.(map[string]interface{}) {
+			if !builtInContexts[wfName] {
```

Comment:
@huydinhle Why do we only check non-builtin context?

Reply:
hmm Because I thought there are no workflows that is named "*" or "_events".  I thought those two are special cases, maybe my naming isn't right?

Reply:
You are right.  They are used for matching workflow names, I thought you were matching them with context names.